src/index.js

```diff
@@ -1,5 +1,5 @@
 import h from "./h"
 import app from "./app"
-import router from "./router"
+import Router from "./router"
```

Comment:
One solution could be to not ship the router with HyperApp anymore. That way you can import the router like

```jsx
import router from "hyperapp-router"

// or

import Router from "hyperapp-router"

// or

import RoUTer from "hyperapp-router"
```